src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -908,11 +1032,15 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         labelMax = qt.QLabel("Max", self)
         labelMax.setAlignment(qt.Qt.AlignHCenter)
         labelMax.setFont(miniFont)
-        rangeLayout.addWidget(labelMin, 0, 0)
-        rangeLayout.addWidget(labelMax, 0, 1)
-        rangeLayout.addWidget(self._minValue, 1, 0)
-        rangeLayout.addWidget(self._maxValue, 1, 1)
-        rangeLayout.addWidget(self._autoButtons, 2, 0, 1, -1, qt.Qt.AlignCenter)
+        rangeLayout.addWidget(labelMin, 0, 1)
+        rangeLayout.addWidget(labelMax, 0, 3)
+        rangeLayout.addWidget(self._minValue, 1, 1)
+        rangeLayout.addWidget(self._maxValue, 1, 3)
+        rangeLayout.setColumnStretch(0, 1)
+        rangeLayout.setColumnStretch(1, 2)
+        rangeLayout.setColumnStretch(2, 1)
+        rangeLayout.setColumnStretch(3, 2)
+        rangeLayout.setColumnStretch(4, 1)
```

Comment:
```suggestion
        rangeLayout.setColumnStretch(4, 1)
```

Why not keeping the min/max with the other widgets on the left side when enlarging the colormap dialog?
Else, to me it would be best to center everything.

Reply:
To me it have to be aligned as much as possible to the min and max location of the plot.

It's the result of many tries.

If you want to change it, i let you play with.